packages/react/src/components/List/List.stories.tsx

```diff
@@ -16,29 +16,28 @@ const decorators = [
 
 export default {
   title: 'Komponenter/List',
-  component: List.Root,
+  component: List.Ordered,
```

Comment:
Wrong component was used? The props/args table is generated from this.
```suggestion
  component: List.Unordered,
```